packages/client-core/src/admin2/components/channel/index.tsx

```diff
@@ -0,0 +1,107 @@
+*/
+
+import { PopoverState } from '@etherealengine/client-core/src/common/services/PopoverState'
+import { ChannelType } from '@etherealengine/common/src/schema.type.module'
+import { useHookstate } from '@etherealengine/hyperflux'
+import Button from '@etherealengine/ui/src/primitives/tailwind/Button'
+import Input from '@etherealengine/ui/src/primitives/tailwind/Input'
+import Text from '@etherealengine/ui/src/primitives/tailwind/Text'
+import React, { useEffect, useRef } from 'react'
+import { useTranslation } from 'react-i18next'
+import { HiMagnifyingGlass, HiPlus, HiTrash } from 'react-icons/hi2'
+import AddEditChannelModal from './AddEditChannelModal'
+import ChannelTable from './ChannelTable'
+import RemoveChannelModal from './RemoveChannelModal'
+
+export default function Channels() {
+  const { t } = useTranslation()
+  const search = useHookstate({ local: '', query: '' })
+  const debouncedSearchQueryRef = useRef<ReturnType<typeof setTimeout>>()
+
+  const selectedChannels = useHookstate<ChannelType[]>([])
+
+  useEffect(() => clearTimeout(debouncedSearchQueryRef.current), [])
+
+  return (
+    <>
+      <div>
+        <Text fontSize="xl" className="mb-6">
+          {t('admin:components.channel.channels')}
+        </Text>
+        <div className="mb-4 flex justify-between">
+          <Input
+            placeholder={t('common:components.search')}
+            value={search.local.value}
+            onChange={(event) => {
+              search.local.set(event.target.value)
+
+              if (debouncedSearchQueryRef) {
+                clearTimeout(debouncedSearchQueryRef.current)
+              }
+
+              debouncedSearchQueryRef.current = setTimeout(() => {
+                search.query.set(event.target.value)
+              }, 100)
+            }}
+            className="dark:bg-[#1A1B1E]"
+            containerClassname="w-1/5 block"
+            icon={<HiMagnifyingGlass />}
+          />
+          <div className="flex w-1/6 gap-4">
+            {selectedChannels.length > 0 && (
+              <div>
+                <Button
+                  startIcon={<HiTrash />}
+                  variant="danger"
+                  size="small"
+                  fullWidth
+                  onClick={() => {
+                    PopoverState.showPopupover(<RemoveChannelModal channels={selectedChannels.value} />)
+                  }}
+                >
+                  {t('admin:components.channel.removeChannels')}
+                </Button>
+              </div>
+            )}
+            <div className="ml-auto">
+              <Button
+                startIcon={<HiPlus />}
+                size="small"
+                fullWidth
+                onClick={() => {
+                  PopoverState.showPopupover(<AddEditChannelModal />)
+                }}
+              >
+                {t('admin:components.channel.createChannel')}
+              </Button>
+            </div>
+          </div>
+        </div>
```

Comment:
![image](https://github.com/EtherealEngine/etherealengine/assets/52322531/b0287ae5-7f73-4704-9074-558859f1ab8b)
Since there is ample space to the left, we can try to avoid wrapping.

Reply:
![image](https://github.com/EtherealEngine/etherealengine/assets/55396651/2a13642e-f7aa-486a-89f1-d215d77f2af7)
implemented!